addons/point_of_sale/static/src/xml/pos.xml

```diff
@@ -1553,7 +1553,7 @@
                     </tr>
                 </t>
                 <tr>
-                    <t t-if="order.get_total_discount() > 0">
+                    <t t-if="order.get_total_discount()">
```

Comment:
I agree this look better, but this is totally out of scope for this task, could you remove this out of the commit?

Reply:
hello, @switch87 
if we remove this then not need to use Math.abs() and Math.sign() in models.js https://github.com/odoo/odoo/pull/31054/commits/a51cf9496b7e591c627811a8ef5f09e53019b955#diff-1c14accbe350f7a177b3e81e5846861bR2506
I have added to show returned product price with a previously applied discount minus.
only thing is left now is, replace "Clear" to "+/-" is it ok?
Thanks

Reply:
@mgh-odoo can you record an example pre- and post-commit on runbot of this? The task was indeed as simple as replacing this key apparently. But if this extra work is indeed an issue that should be fixed, you can split the commit into one  [IMP] and one [FIX] and your work would not be for nothing :)

Reply:
hello, @switch87 
I changed commit based on your suggestion and 
also recorded a video about [pre-commit](https://drive.google.com/open?id=1dKH5g7RZARmsN_0p43FaGNw7mGDNvyqf) and [post-commit](https://drive.google.com/open?id=17xgvw3Ov0HqViKy7iaLnyGKG0lMJ44_x) 
if it's out of scope, let me know I'll remove that one :)

Reply:
@mgh-odoo yes it's out of scope for this task. I think it is not a bad idea, so make it a new pr for it and ask @lap-odoo for her opinion.

Reply:
@switch87  done. :+1: